components/brave_ads/browser/component_updater/resource_component.cc

```diff
@@ -119,7 +119,8 @@ void ResourceComponent::LoadManifestCallback(const std::string& component_id,
                                              const std::string& json) {
   VLOG(8) << "Manifest JSON: " << json;
 
-  std::optional<base::Value::Dict> dict = base::JSONReader::ReadDict(json);
+  std::optional<base::Value::Dict> dict =
+      base::JSONReader::ReadDict(json, base::JSON_PARSE_CHROMIUM_EXTENSIONS);
```

Comment:
`JSON_PARSE_RFC` and elsewhere. Thanks

Reply:
Actually after discussing with @aseren the default was always JSON_PARSE_CHROMIUM_EXTENSIONS so for this CR bump lets leave as-is, and we will raise an issue to change once this merges.

Reply:
filed an issue for this: https://github.com/brave/brave-browser/issues/49949